e2e/env/common.go

```diff
@@ -91,10 +98,49 @@ func init() {
 			versionBundleVersion = vbs[len(vbs)-2].Version
 		}
 	}
+
 	err := os.Setenv(EnvVarVersionBundleVersion, VersionBundleVersion())
 	if err != nil {
 		panic(err)
 	}
+
+	clusterID := os.Getenv("CLUSTER_NAME")
```

Comment:
This was here from the beginning :see_no_evil:  but we just recalculated the ID every time in the ClusterID func. Now it's also possible to set entire cluster ID with `CLUSTER_NAME` env var.